main.go

```diff
@@ -132,18 +134,26 @@ func main() {
 	ctrl.SetLogger(klogr.New())
 
 	mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
-		Scheme:                     scheme,
-		MetricsBindAddress:         metricsBindAddr,
+		Scheme: scheme,
+		Metrics: server.Options{
+			BindAddress: metricsBindAddr,
+		},
 		LeaderElection:             enableLeaderElection,
 		LeaderElectionID:           "controller-leader-election-rancher-turtles",
 		LeaderElectionResourceLock: resourcelock.LeasesResourceLock,
 		LeaseDuration:              &leaderElectionLeaseDuration,
 		RenewDeadline:              &leaderElectionRenewDeadline,
 		RetryPeriod:                &leaderElectionRetryPeriod,
-		SyncPeriod:                 &syncPeriod,
-		ClientDisableCacheFor: []client.Object{
-			&corev1.ConfigMap{},
-			&corev1.Secret{},
+		Client: client.Options{
+			Cache: &client.CacheOptions{
+				DisableFor: []client.Object{
+					&corev1.ConfigMap{},
+					&corev1.Secret{},
+				},
+			},
+		},
+		Cache: cache.Options{
+			SyncPeriod: &syncPeriod,
```

Comment:
Why is the sync period moved to a new location? is related to the controller-runtime bump?

Reply:
It is related to controller-runtime